fastlane_bot/events/exchanges/uniswap_v3.py

```diff
@@ -30,7 +30,12 @@ def add_pool(self, pool: Pool):
         self.pools[pool.state["address"]] = pool
 
     def get_abi(self):
-        return UNISWAP_V3_POOL_ABI if self.exchange_name not in [PANCAKESWAP_V3_NAME, AGNI_V3_NAME] else PANCAKESWAP_V3_POOL_ABI
+        if self.exchange_name == FUSIONX_V3_NAME:
+            return FUSIONX_V3_POOL_ABI
+        elif self.exchange_name not in [PANCAKESWAP_V3_NAME, AGNI_V3_NAME]:
+            return UNISWAP_V3_POOL_ABI
+        else:
+            return PANCAKESWAP_V3_POOL_ABI
```

Comment:
not specific to those changes here but this is a case where you should check for all possible conditions and have an else at the end that raises and "unknown exchange"

otherwise errors can percolate much further and may be harder to detect